src/DynamoCoreWpf/UI/GuidedTour/Step.cs

```diff
@@ -397,7 +411,7 @@ public void UpdateLocation()
         /// <summary>
         /// Update the Location of Popups only when they are located over the Library (HostPopupInfo.WindowName = LibraryView)
         /// </summary>
-        public void UpdateLibraryPopupsLocation()
+        internal void UpdateLibraryPopupsLocation()
```

Comment:
This is API breaking but I think we informed community that UI level API breaks could be introduced by our team 👍🏻